finder.ts

```diff
@@ -42,6 +44,8 @@ export function finder(input: Element, options?: Partial<Options>) {
     optimizedMinLength: 2,
     threshold: 1000,
     maxNumberOfTries: 10000,
+    timeoutMs: -1,
+    start: new Date(),
```

Comment:
To me start seems more like an internal constant that does not need to be exposed through options. Can we set the start const globally?

Reply:
Good call!